osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/domain/entities/Location.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2021 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.kafka.da.domain.entities;
+
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+
+import org.opensmartgridplatform.shared.domain.entities.AbstractEntity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class Location extends AbstractEntity {
+
+    @OneToMany(mappedBy = "location", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Lazy loading was failing, besides feeders are always needed when loading a location.

Reply:
Just changing the `FetchType` to `LAZY` without further changes would indeed fail I think.
When you make relationships lazy you need to get explicit about your fetch strategies outside the entity itself.
Here is an article where somebody more knowledgeable about JPA and Hibernate than I tries to explain things:
https://vladmihalcea.com/eager-fetching-is-a-code-smell/
If you feel confident this case would fall under exceptions to the general warnings, because the feeders are needed in all business use cases for the Location, I'll follow you. In the code as it is right now it would probably (?) not be an issue, but we need to be careful when for instance other relations would be mapped or similar changes.